src/layouts/ModelizerDrawLayout.vue

```diff
@@ -92,6 +112,19 @@ async function initView() {
 onMounted(() => {
   initView();
 });
+
+onUnmounted(() => {
+  pluginInitSubscription.unsubscribe();
+});
+
+/**
+ * Rearrange components on click.
+ */
+async function arrangeComponentsPosition() {
+  await data.plugin.arrangeComponentsPosition();
+  data.plugin.draw(drawTarget);
+}
+
```

Comment:
Missing `changelog.md` update in another commit and dependencies update in another commit too.

Reply:
changelog: done
dependencies: done

Reply:
e2e tests: done